deploy/test/test_cases/TEST_ID_17_helm_job_deploys_successfully.sh

```diff
@@ -14,13 +14,15 @@ popd
 # Deploy app to test against
 deploy_helm_app
 
-# Check for Job completion
+# Wait for Job completion
 helm_chart_name="secrets-provider"
-$cli_with_timeout wait --for=condition=complete job/$helm_chart_name
+$cli_with_timeout "get job/$helm_chart_name -o=jsonpath='{.status.conditions[*].type}' | grep Complete"
```

Comment:
This line repeat 8 times in this code. I think put some wait command in utils may improve the code. 
```
wait_until_job_complete() {
     $cli_with_timeout "get job/$helm_chart_name -o=jsonpath='{.status.conditions[*].type}' | grep Complete"
}
```

Reply:
I don't mind it being repeated over and over again because it is only one line and so I don't see much value in throwing it in a function